x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/field_components/drag_and_drop_text_list.tsx

```diff
@@ -162,31 +161,22 @@ function DragAndDropTextListComponent({
                                 return (
                                   <EuiFlexGroup gutterSize="none" alignItems="center">
                                     <EuiFlexItem>
```

Comment:
Since there's only one flex item now, you can probably remove this flex group/item wrapper too.

Reply:
Nice spot!